### PR TITLE
PMM-9322 Added support of PSMDB 1.11.0

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/pmm.2.27.0.pmm-server.json [hunk 1]
```diff
@@ -42,6 +42,12 @@
             "image_hash": "1b6b5dda3c9ec50128f8e4fd0a9e07feb16bb262b92f8d31f09fe19df51b069a",
             "status": "recommended",
```
Comment (tplavcic) [inline sources/pmm.2.27.0.pmm-server.json:43]: Can two versions be recommended at the same time?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: sources/pmm.2.27.0.pmm-server.json (hunk 1 above)
Comment (BupycHuk) [reply sources/pmm.2.27.0.pmm-server.json]: Fixed

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by beatahandzelova — APPROVED** [review]

**Pull request merged**